pull_images_4.2/use_downstream_operatorresource.sh

```diff
@@ -0,0 +1,71 @@
+ 
+#!/usr/bin/env bash
+#!/bin/bash
+
+# Source: Openshift-qe
+# https://github.com/openshift-qe/v3-testfiles/blob/master/logging/tools/use_downstream_operatorresource.sh#L68
+
+NAMESPACE=${1:-redhat-operators-art}
+
+
+function updateCluster()
+{
+echo "# disable redhat-operators.yaml"
+cat <<EOF >disbale_redhat_operator.yaml
```

Comment:
I think we could create this as static yaml files in top level folder but not holding it for now.